phoenicis-dist/src/launchers/phoenicis

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+
+/usr/share/phoenicis/PhoenicisPlayOnLinux "$@"
```

Comment:
Add newline at EOF.

Reply:
I think it is now updated isn't it?

Reply:
No. Check the diff. It shows this red symbol.

Reply:
Ok. Fixed.